src/test/java/net/snowflake/client/jdbc/structuredtypes/ResultSetStructuredTypesLatestIT.java

```diff
@@ -663,13 +664,36 @@ public void testColumnTypeWhenStructureTypeIsDisabled() throws Exception {
 
   @Test
   @ConditionalIgnoreRule.ConditionalIgnore(condition = RunningOnGithubAction.class)
-  public void testColumnTypeWhenStructureTypeIsEnabled() throws Exception {
+  public void testColumnTypeAndFieldsWhenStructureTypeIsEnabled() throws Exception {
```

Comment:
I think we don't use "enabled" in this context any more?

Reply:
Fixed